### PR TITLE
Code example fixes and unit tests

### PR DESCRIPTION
**Pull request by ghost**
#1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,18 +13,18 @@ Start using it straight away. (Following example assumes that it was installed v
 ```php
 <?php
 
-require 'path/to/vendor/autoload.php';
+require __DIR__.'/vendor/autoload.php';
```
Comment (jabranr) [inline README.md]: Since this is just a README therefore the path as set to generic/pseudo type. I would recommend reverting this.

**Review by jabranr — CHANGES_REQUESTED** [review]
Thank you for this. Just one quick change for you.

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: No problem and glad to help! I just added it back instead of trying to get rid of that commit.

**Review by jabranr — APPROVED** [review]

**Review by jabranr — APPROVED** [review]

**Comment by jabranr** [comment]
Resolves issue #1

**Pull request merged**